src/sass/claims-status.scss

```diff
@@ -628,41 +612,45 @@ h1:focus {
 
 .claims-list-sort {
   margin: 2em 0;
+}
 
-  label, select {
-    display: inline-block;
-  }
+.claims-list-sort-label,
+.claims-list-sort-select {
+  display: inline-block;
+}
 
-  label {
-    margin-right: 1.5rem;
-  }
+.claims-list-sort-label {
+  margin-right: 1.5rem;
+}
 
-  @media (max-width: $medium-screen) {
-    label {
-      display: block;
-      margin: 0;
-    }
+@media (max-width: $medium-screen) {
+  .claims-list-sort-label {
+    display: block;
+    margin: 0;
+  }
 
-    select {
-      display: block;
-      margin: 1.4rem 0;
-    }
+  .claims-list-sort-select {
+    display: block;
+    margin: 1.4rem 0;
   }
 }
 
 .claims-list-alert {
   margin-bottom: 1em;
   margin-top: 0;
+
   &.usa-alert {
     background-size: 3.4rem;
     background-position: 1rem 1.9rem;
   }
-  > .usa-alert-body {
-    padding-left: 3.7rem;
-  }
-  .va-alert-close .fa {
-    font-size: 1.1em !important;
-  }
+}
+
+.claims-list-alert-body {
```

Comment:
I think you could get away with not changing this (i.e. not creating a class for it and leaving it under `.claims-list-alert` like before). For reusable widgets and patterns like alerts and the subway map, I think it's ok to have one level of nesting in the styles if we just need to override a small portion of the default styling.

Reply:
Thanks @jbalboni, good point, reverted